airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -300,7 +300,9 @@ def get_json_schema(self) -> Mapping[str, Any]:
         loader = ResourceSchemaLoader(package_name_from_class(self.__class__))
         schema = loader.get_schema("ads_insights")
         if self._fields:
-            schema["properties"] = {k: v for k, v in schema["properties"].items() if k in self._fields + [self.cursor_field]}
+            # 'date_stop' and 'account_id' are also returned by default, even if they are not requested
+            custom_fields = self._fields + [self.cursor_field, "date_stop", "account_id", "ad_id"]
```

Comment:
Let's de duplicate fields, because user can choose one of default fields during configuration

Reply:
'custom_fields' is just used to check if field from schema in is list of custom fields. Thus at the end there are no duplicates